### ISSUE TITLE
how to link generate in every row of swift dataTables

### ISSUE THREAD

**Issue by NupendraVerma** [pavankataria/SwiftDataTables#11]
how to generate link in each row of data table 
and I want to remove search bar in the data table how??

**Comment by pavankataria** [comment]
Hey there, this would require the api allowing custom cells to be returned, and then you could add your link in the form of an attributed text with appropriate openApp url handling. 

I'm a bit tied up atm, but a PR is totally welcomed :)

**Comment by pavankataria** [comment]
> I want to remove search bar in the data table how??
🎉 🎉 @NupendraVerma  This is now available as of version 0.6.22 🎉 🎉 https://github.com/pavankataria/SwiftDataTables/releases/tag/0.6.22

Set the `shouldShowSearchSection` property to false in the `DataTableConfiguration` file and inject that into your data table initialiser and you're good to go. The search view will not be added.

You can also opt to the use the new delegate method and return false to achieve the same thing. 
```Swift
@objc optional func shouldShowSearchSection(in dataTable: SwiftDataTable) -> Bool
```

Let me know how it goes. 🎉

**Comment by pavankataria** [comment]
> I want to remove search bar in the data table how??
🎉 🎉 @NupendraVerma  This is now available as of version 0.6.22 🎉 🎉 https://github.com/pavankataria/SwiftDataTables/releases/tag/0.6.22

Set the `shouldShowSearchSection` property to false in the `DataTableConfiguration` file and inject that into your data table initialiser and you're good to go. The search view will not be added.

You can also opt to the use the new delegate method and return false to achieve the same thing. 
```Swift
@objc optional func shouldShowSearchSection(in dataTable: SwiftDataTable) -> Bool
```

Let me know how it goes. 🎉
